x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/existence/FrozenExistenceDeciderIT.java

```diff
@@ -72,10 +74,14 @@ protected Settings nodeSettings(int nodeOrdinal, Settings otherSettings) {
 
     @Override
     protected Collection<Class<? extends Plugin>> nodePlugins() {
-        return List.of(BlobCachePlugin.class, LocalStateAutoscalingAndSearchableSnapshotsAndIndexLifecycle.class);
+        return List.of(
+            BlobCachePlugin.class,
+            LocalStateAutoscalingAndSearchableSnapshotsAndIndexLifecycle.class,
+            SnapshotLifecycle.class,
+            Ccr.class
```

Comment:
Could you please explain why it needs this dependencies? CCR looks especially surprising for me

Reply:
@idegtiarenko Sure! SLM is needed due to #98184 and CCR is needed since `WaitForNoFollowersStep` now requires it since #94510